methods.py

```diff
@@ -46,13 +44,13 @@ def instrFilter(x,y):
         outputdict[kv] = outputlist
     return outputdict
 
+
 def subsetCreation(x,y):
     """Takes output from instrFilter as its x-arg and the desired category as y. Returns a list of spectra"""
     output = x.get(y)
     return output

```

Comment:
This is not what instrFilter, currently outputs. This will return a dictionary with the number of times each names occurs, but not the spectra that belong to this type.